sdk/storage/azblob/blob/constants.go

```diff
@@ -18,6 +18,8 @@ const (
 
 	// DefaultDownloadBlockSize is default block size
 	DefaultDownloadBlockSize = int64(4 * 1024 * 1024) // 4MB
+
+	defaultConcurrency = 5
```

Comment:
We use the same value in `/internal/shared/batch_transfer.go`.  Can we move this there and consolidate them?